test/test_unary_ufuncs.py

```diff
@@ -910,6 +910,7 @@ def test_helper(min, max):
     @onlyCPU
     @slowTest
     @dtypes(torch.float)
+    @unittest.skipIf(True, "Insufficient memory on linux.(2|4)xlarge")
```

Comment:
@suo @malfet any opinions on whether we should skip this test entirely, or move the slow test to a 9xlarge just to have enough memory for this test?